dev/com.ibm.ws.jaxws.2.2.webcontainer_fat_extended/fat/src/com/ibm/ws/jaxws/fat/HandlerChainTest.java

```diff
@@ -36,6 +36,9 @@
 import componenttest.topology.impl.LibertyServer;
 import componenttest.topology.utils.HttpUtils;
 
+/**
+ *
```

Comment:
Will you add a comment to this test suite explain what the tests in the suite do?

```suggestion
 * This suite tests the client side handler chain, including testing the expected behavior when the client parses a
 * response containing additional unexpected elements in the incoming SOAP response.
```